docs/content/gameobject/reading-and-writing-component-data.md

```diff
@@ -28,202 +28,134 @@ We use the following `Health` schema component for all the code examples in this
 package improbable.examples;
 
 type IntMessage {
-
   int32 value = 1;
-
 }
 
 type Empty {}
 
 component Health {
-
   id = 12345;
-
   int32 current_health = 1;
-
   event IntMessage health_changed;
-
   command Empty set_health(IntMessage);
-
 }
-
 ```
 
-## How to read the component state
+## How to read component properties
```

Comment:
I think component data was clearer. Events and commands are properties of a component in some sense.
Also this is the first time I've ever heard this referred to as properties.

Reply:
See https://docs.improbable.io/reference/13.2/shared/schema/reference